provisioning-workflows/IIB/IIBv9/jcl/EnvironmentFileTemplate.txt

```diff
@@ -0,0 +1,224 @@
+#                     Note that this setting requires that DB2 is
+#                     configured to accept mixed byte data. This
+#                     setting should be used when the customer
+#                     wants data to be stored in the configured
+#                     DB2 EBCDIC mixed byte code page.
+#          MQSI_DB2_CONVERSION=LOCAL
+#                     WBIMB uses localConverter identified by
+#                     LC_ALL/LANG settings. This complies to what
+#                     is done on distributed. This setting
+#                     requires that WBIMB and DB2 are using the same
+#                     codepage, otherwise only WBIMB can read DB2
+#                     data correctly, it gets unreadable for other
+#                     ( non-WBIMB ) applications that want to read
+#                     the data.
+#
+# 4.3 DSNAOINI references the component dsnaoini file.
+#     (NO NEED TO CHANGE FROM DEFAULT!)
+#
+export MQSI_DB2_ALWAYS_PREPARE=NO
+export MQSI_DB2_CONVERSION=UNICODE 
+export DSNAOINI=//\'${instance-ZOSMF_PARMLIB}\(${instance-IIB_DSNAOINI}\)\'
+# *****************************************************************
+# 5. Java Settings
+# *****************************************************************
+#
+# 5.1 JAVAHOME contains the root directory of the JAVA install.
+#     e.g. JAVAHOME=/usr/lpp/java/IBM/J1.7
+#  ${instance-IIB_JAVAHOME}
+export JAVAHOME=${instance-IIB_JAVAHOME} 
+# *****************************************************************
+# 6. IBM Integration Bus Settings
+# *****************************************************************
+#
+# 6.1 _BPX_BATCH_SPAWN
+#     (MUST NOT CHANGE FROM DEFAULT!)
+#
+# 6.2 MQSI_MC_MESSAGES determines if messages should appear in
+#     mixed case or upper case.
+#     e.g. MQSI_MC_MESSAGES=YES for mixed case
+#          MQSI_MC_MESSAGES=NO for upper case
+#
+# 6.3 MQSI_COMMAND_DATABASE_ECHO if defined, mqsi commands
+#     display information when creating DB2 tables/indexes.
+#
+# 6.4 MQSI_COMMAND_ZOS_MQ_ECHO if defined, mqsi commands display
+#     information returned from the MQ command server when
+#     creating/deleting queues.
+#
+# 6.5 MQSI_COMMAND_ZOS_MQ_ECHO_RC if defined, mqsi commands display
+#     reason and return codes from the MQ command server when
+#     creating/deleting queues.
+#
+# 6.6 MQSI_DEPLOY_PROGRESS if defined, shows deployment progress
+#     by the integration server.
+#
+# 6.7 STEPLIB
+#     (MUST NOT CHANGE FROM DEFAULT!)
+#
+# 6.8 MQSI_COMMAND_ZOS_ERROR_RC if defined, mqsi commands return
+#     this value as the return code if an error occurs.
+#
+# 6.9 MQSI_FAD current FAD level.
+#
+export _BPX_BATCH_SPAWN=NO
+export MQSI_MC_MESSAGES=NO
+export MQSI_COMMAND_DATABASE_ECHO=1
+export MQSI_COMMAND_ZOS_MQ_ECHO=1
+export MQSI_COMMAND_ZOS_MQ_ECHO_RC=1
+export MQSI_DEPLOY_PROGRESS=1
+export STEPLIB=SYS3.${instance-IIB_DBSRV_NAME}.SDSNEXIT:SYS3.${instance-IIB_DBSRV_NAME}.SDSNLOAD:SYS3.${instance-IIB_DBSRV_NAME}.SDSNLOD2   
```

Comment:
SYS3 is assumed here, but should probably made variable. Just like &DB2HLQ in /jcl/UPDSTC.jcl ?